runtime/web_worker.rs

```diff
@@ -589,7 +586,7 @@ mod tests {
 
     let mut handle = handle_receiver.recv().unwrap();
 
-    let msg = json!("hi").to_string().into_boxed_str().into_boxed_bytes();
+    let msg = vec![34, 2, 104, 105].into_boxed_slice(); // "hi" encoded
```

Comment:
Consider using serde_v8 to encode the value, here and below. The wire format isn't stable.

Reply:
Sorry, yes. I suppose that makes it inconvenient to call out to. Consider it a very light suggestion then.

Reply:
I added some TODO comments here, if accessing `Deno.core.serialize()` from rust would become easier this can be fixed; also depends on how often it breaks due to v8 internals changing